### PR TITLE
re-enable tests

### PR DESCRIPTION
**Pull request by conorbros**

### CODE REVIEW

File: shotover-proxy/tests/cassandra_int_tests/prepared_statements.rs [hunk 1]
```diff
@@ -73,26 +73,26 @@ async fn select_cross_connection<Fut>(
 async fn use_statement(session: &CassandraConnection) {
     // Create prepared command with the correct keyspace
     run_query(session, "USE test_prepare_statements;").await;
-    let _prepared = session.prepare("INSERT INTO table_1 (id) VALUES (?);");
+    let prepared = session.prepare("INSERT INTO table_1 (id) VALUES (?);");
 
     // change the keyspace to be incorrect
     run_query(session, "USE test_prepare_statements_empty;").await;
 
     // TODO: Somehow executing the query fails when run through shotover but succeeds when run directly against cassandra
```
Comment (rukai) [inline shotover-proxy/tests/cassandra_int_tests/prepared_statements.rs:81]: huh, I have no idea what we changed to fix this test.
Considering it is fixed though, we should remove this TODO.

File: shotover-proxy/tests/cassandra_int_tests/prepared_statements.rs [hunk 2]
```diff
@@ -80,28 +80,24 @@ async fn select_cross_connection<Fut>(
...
-    // );
-
-    // // observe that the query succeeded despite the keyspace being incorrect at the time.
-    // assert_query_result(
-    //     session,
-    //     "SELECT id FROM test_prepare_statements.table_1 WHERE id = 358;",
-    //     &[&[ResultValue::Int(358)]],
-    // )
-    // .await;
+    assert_eq!(
```
Comment (rukai) [inline shotover-proxy/tests/cassandra_int_tests/prepared_statements.rs:90]: Oh we lost this comment: `// observe query completing against the original keyspace without errors`

File: shotover-proxy/tests/cassandra_int_tests/prepared_statements.rs [hunk 3]
```diff
@@ -80,28 +80,24 @@ async fn select_cross_connection<Fut>(
...
+        session.execute_prepared(&prepared, 358),
+        Vec::<Vec<ResultValue>>::new()
+    );
+
+    assert_query_result(
```
Comment (rukai) [inline shotover-proxy/tests/cassandra_int_tests/prepared_statements.rs:95]: And this one `// observe that the query succeeded despite the keyspace being incorrect at the time.`

File: shotover-proxy/tests/cassandra_int_tests/prepared_statements.rs (hunk 2 above)
Comment (conorbros) [reply shotover-proxy/tests/cassandra_int_tests/prepared_statements.rs:90]: Oh thought that was just part of the TODO

**Review by rukai — APPROVED** [review]
lgtm

**Review by benbromhead — APPROVED** [review]

**Review by XA21X — APPROVED** [review]

**Pull request merged**